cmake/verify-toolchain.cmake

```
@@ -16,7 +16,11 @@
 # FORMAT=json: Print the output as a json formatted string, useful for Python
 
 # This is the minimum required Zephyr-SDK version which supports CMake package
-set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.11.3)
+if("${ARCH}" STREQUAL "arm64")
+  set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.12.4)
+else()
+  set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.11.3)
```

As per https://github.com/zephyrproject-rtos/zephyr/pull/34064#discussion_r633641075

please change this directly to `0.12`
```suggestion
  set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.12)
```